packages/circuits/semaphore.circom

```diff
@@ -1,90 +1,25 @@
-pragma circom 2.0.0;
+pragma circom 2.1.5;
 
-include "../node_modules/circomlib/circuits/poseidon.circom";
-include "./tree.circom";
+include "babyjub.circom";
+include "poseidon.circom";
+include "binary-merkle-root.circom";
 
-template CalculateSecret() {
-    signal input identityNullifier;
-    signal input identityTrapdoor;
+template Semaphore(MAX_DEPTH) {
+    signal input privateKey;
```

Comment:
What about using the name `secret` or `secretScalar` instead of `privateKey` because it can be confusing with the private key of the identity, and they are not the same.

Reply:
Please see: https://github.com/semaphore-protocol/semaphore/pull/480/commits/076e5de03c3646590e33f830dd75905e6fc0103d